applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1009,8 +1009,8 @@ static void on_device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 			bt_foreach_bond(BT_ID_DEFAULT, bond_connect, (void *)addr);
 		}
 		return;
-	} else if ((type == BT_GAP_ADV_TYPE_ADV_IND || type == BT_GAP_ADV_TYPE_EXT_ADV) &&
-		   (bonded_num < CONFIG_BT_MAX_PAIRED)) {
+	} else if ((type == BT_GAP_ADV_TYPE_ADV_IND || type == BT_GAP_ADV_TYPE_EXT_ADV || (type == BT_GAP_ADV_TYPE_SCAN_RSP)) &&
+	   		(bonded_num < CONFIG_BT_MAX_PAIRED)) {
```

Comment:
Either add parenthesis to all of the 'type == BT_GAP...' or remove the ones for your addition, so that it looks more consistent. There is also the formatting, we use clang for out files, and the line shouldn't be more than 100 characters. I've posted the correct formatting below
```suggestion
	} else if ((type == BT_GAP_ADV_TYPE_ADV_IND || type == BT_GAP_ADV_TYPE_EXT_ADV ||
		    type == BT_GAP_ADV_TYPE_SCAN_RSP) &&
		   (bonded_num < CONFIG_BT_MAX_PAIRED)) {
```